lms/static/scripts/frontend_apps/components/dashboard/DashboardActivityFilters.tsx

```diff
@@ -6,24 +6,24 @@ import { useConfig } from '../../config';
 import { useAPIFetch } from '../../utils/api';
 
 export type DashboardActivityFiltersProps = {
-  selectedCourses: Course[];
-  onCoursesChange: (newCourses: Course[]) => void;
-  selectedAssignments: Assignment[];
-  onAssignmentsChange: (newAssignments: Assignment[]) => void;
-  selectedStudents: Student[];
-  onStudentsChange: (newStudents: Student[]) => void;
+  selectedCourseIds: string[];
```

Comment:
There is an oddity here that course and assignment IDs are represented as integers in API responses and the types that are passed around, but we then convert them to strings here.